### DISCUSSION TITLE
Search for items without a Category / Tag / Tool

### DISCUSSION THREAD

**Discussion by leitenky** [mealie-recipes/mealie#4701] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

I am trying to clean my catalog of recipes  up, currently you can search for categories/tags/tools/food for "has any" option under the 4 options. I would like to filter for ones that have "None" to any of the attributes so I can review and update those recipes 

### Please provide a concise description of the feature that would resolve your issue.

When setting filters in the search filters for categories/tags/tools/foods, have an option to have "None" so recipes can be review

### Please consider and list out some caveats or tradeoffs made in your design decision

None that I can think of.

### Additional Information

- [X] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by michael-genson** [comment]
For cleanup purposes this is already available. On the data management page you can sort by category/tag and it will put "no tags" (or categories) at the top (or bottom, depending on the sort direction)

Similar discussion here: https://github.com/mealie-recipes/mealie/discussions/4626
